README.md

```diff
@@ -126,14 +133,32 @@ When the File Watcher service starts, it will look into the folder configured as
 - `game_title`: The name of the game
 - `localizations`: Localization management for title and description. Not working for now
 
-In the Watch folder, you will need to create a `thumbnails` folder containing all the episodes thumbnails named by the episode number and in png format 1280x720p
+In the serie's folder, you will need to create a `thumbnails` folder containing all the episodes thumbnails named by the episode number and in png format 1280x720p.
+
+An example of file structure would be:
+
+```
+--Watch/
```

Comment:
It was quite hard to know how to layout the files without watching the videos, so I added this example.